packages/sparqlee/package.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Some things are missing still:
* `repository`
* `publishConfig`
* `sideEffects`
* `bugs`
* `homepage`

Reply:
Perhaps we also want to re-order the entries to be aligned with the other package.json files in other packages.

Reply:
For `sideEffects` I'm not sure... we do have some global vars. But I do think we can remove them if not used?

Reply:
Aha, good point. Let's not add `sideEffects` then, but make an issue to investigate how to enable it.